packages/driver/cypress/e2e/e2e/origin/commands/misc.cy.ts

```diff
@@ -219,7 +219,7 @@ it('verifies number of cy commands', () => {
     'focused', 'get', 'contains', 'root', 'shadow', 'within', 'request', 'session', 'screenshot', 'task', 'find', 'filter', 'not',
     'children', 'eq', 'closest', 'first', 'last', 'next', 'nextAll', 'nextUntil', 'parent', 'parents', 'parentsUntil', 'prev',
     'prevAll', 'prevUntil', 'siblings', 'wait', 'title', 'window', 'document', 'viewport', 'server', 'route', 'intercept', 'origin',
-    'mount',
+    'mount', 'getAllLocalStorage', 'clearAllLocalStorage', 'getAllSessionStorage', 'clearAllSessionStorage',
```

Comment:
With @BlueWinds changes, several of these are longer added as `commandFns` but as `queryFns` and `ng` was removed. How is this working??

Reply:
I think Blue's changes are only in the v12 branch, right? This is targeting `develop`.